src/fal/fal_script.py

```diff
@@ -26,9 +26,10 @@ def exec(self, context, faldbt: FalDbt):
         sys.path.append(local_path)
 
         with open(self.path) as file:
-            a_script = file.read()
+            script_ast = compile(file.read(), file.name, "exec", dont_inherit=True)
```

Comment:
why do we need to do this?

Reply:
I think this and all the results stuff can actually go in a different PR. I will do that.